fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintainer.java

```diff
@@ -428,7 +429,7 @@ private static CompletableFuture<Void> rebalancePartitions(final FDBDatabaseRunn
                                                         continuation.set(newContinuation);
                                                         return true;
                                                     }
-                                                })))), state.context.getExecutor());
+                                                }))), repartitioningLogMessages.logMessages), state.context.getExecutor());
```

Comment:
I'm not sure which version of `runAsync` will be triggered by adding the `repartitioningLogMessages.logMessages)`. Would it be one that expects a lambda that provides an additional log items (as an even list of objects)?